lib/Utils.vala

```diff
@@ -31,6 +31,8 @@ namespace Gala {
 
         private static AppCache app_cache;
 
+        private static Gtk.IconTheme icon_theme = new Gtk.IconTheme ();
+
         static construct {
```

Comment:
This works for apps that ship their own icon, but doesn't work for things that use system icons like Files. We can hardcode though to fix it:
```suggestion
        private static Gtk.IconTheme icon_theme;

        static construct {
            icon_theme = new Gtk.IconTheme ();
            icon_theme.set_custom_theme ("elementary");
```

Reply:
Thank you :)